etc/kayobe/environments/ci-multinode/network-allocation.yml

```diff
@@ -0,0 +1,7 @@
+admin_ips:
```

Comment:
This file should be generated by terraform. Although I thought we also needed some other config for the admin network - CIDR, gateway, etc.

Reply:
Shouldn't that be generated by TF?